Orange/canvas/config.py

```diff
@@ -44,6 +45,11 @@ def init():
     QCoreApplication.setApplicationVersion(version)
     QSettings.setDefaultFormat(QSettings.IniFormat)
 
+    # Hack to make sure the correct plugin search path is added. (Github issue #1143)
+    for path in site.getsitepackages():
+        if path.endswith("site-packages"):
+            QCoreApplication.addLibraryPath(os.path.join(path, "PyQt4", "plugins"))
```

Comment:
> we should append the path to the end instead of the front?

Exactly, we get to honor any user overrides, but if our requirements aren't met in any of the already set paths, the fallback default will fill them in.